src/main/kotlin/com/sourcegraph/cody/inspections/CodyFixHighlightPass.kt

```diff
@@ -137,6 +131,12 @@ class CodyFixHighlightPass(val file: PsiFile, val editor: Editor) :
     for (highlight in myHighlights) {
       highlight.unregisterQuickFix { it.familyName == CodeActionQuickFix.FAMILY_NAME }
 
+      if (highlight.startOffset > document.textLength ||
+          highlight.endOffset > document.textLength ||
+          highlight.startOffset > highlight.endOffset) {
+        break
+      }
+
       val range = document.codyRange(highlight.startOffset, highlight.endOffset)
```

Comment:
Is that running in the EDT?

Reply:
It runs in EDT. I added `@RequiresEdt` on it.